server/utils/formUtils.ts

```diff
@@ -0,0 +1,32 @@
+import { body, Result, ValidationChain, ValidationError, validationResult } from 'express-validator'
+import { Request } from 'express'
+
+export default class FormUtils {
+  static yesNoRadioWithConditionalInputValidationChain({
```

Comment:
this name is long and quite hard to read but is probably as good as can be isn't it

Reply:
I don't have a better name for it, but perhaps could stick it in a `validations` object so it would be accessed as `FormUtils.validations.yesNoRadioWithConditionalInput`?